WordPress/src/main/java/org/wordpress/android/ui/posts/PostSettingsInputDialogFragment.java

```diff
@@ -8,21 +8,19 @@
 import android.text.TextUtils;
 import android.text.TextWatcher;
 import android.view.LayoutInflater;
-import android.view.View;
-import android.widget.EditText;
-import android.widget.TextView;
 
 import androidx.annotation.Nullable;
 import androidx.appcompat.app.AlertDialog;
 import androidx.appcompat.view.ContextThemeWrapper;
 import androidx.fragment.app.DialogFragment;
 
 import com.google.android.material.dialog.MaterialAlertDialogBuilder;
-import com.google.android.material.textfield.TextInputLayout;
 
 import org.wordpress.android.R;
+import org.wordpress.android.databinding.PostSettingsInputDialogBinding;
 import org.wordpress.android.util.ActivityUtils;
 
+
```

Comment:
Minor (🔍): Consider removing this extra line.

Reply:
Fixed.